### ISSUE TITLE
Module not working on the latest version of Playstore

### ISSUE THREAD

**Issue by aiolos01** [P1nGu1n/PlayStoreChangelog#15]
I know this is an ongoing issue but I just wanted to let you know that PlayStoreChangelog isn't working any more (since 8.8.12 I believe). I hope you're still supporting this module as it's very useful.

**Comment by ChaliceChore** [comment]
yes it's not working. i request you to fix this. on redmi note 3 miui 9 nougat.
